mods/ra/rules/structures.yaml

```diff
@@ -570,9 +570,13 @@ DOME:
 		Type: Wood
 	RevealsShroud:
 		Range: 10c0
+		RequiresCondition: !disabled
 		RevealGeneratedShroud: False
+	RevealsShroud@Offline:
+		Range: 5c0
```

Comment:
Nit: I know this won't have any gameplay impact, as it's hidden by the other ones as long as they are enabled, but for performance reasons I'd give this `RequiresCondition: disabled`, to disable it while the other two are enabled (our engine probably does visibility updates for every enabled trait, regardless of whether there's one with higher Range that 'drowns' it).